### PR TITLE
New API documentation

### PR DESCRIPTION
**Pull request by SigismundSchlomo**
## Why
Old api documentation was outdated
## What was done 
Api documentation was improved
Documentation could be checked here 
https://amrosusnode.docs.apiary.io/#introduction/authentication
Also, there is a confluence doc
https://inc4net.atlassian.net/wiki/spaces/AMB/pages/175866027/Ambrosus+node+api

### CODE REVIEW

**Comment by vitalii427** [comment]
As I see API isn't complete yet

File: apiary.apib [hunk 1]
```diff
@@ -1569,6 +1607,169 @@ Fetches a bundle metadata by identifier
...
+                "pagination": {
+                    "hasNext": false,
+                    "next": "eyIkb2lkIjoiNjFhNGE4MWNmNjE2YmEwMDNjY2IyOGMwIn0",
+                    "hasPrevious": false,
+                    "previous": "eyIkb2lkIjoiNjFhNGE4MWNmNjE2YmEwMDNjY2IyOGMwIn0"
+                }
+            }
+
+
+## Group Bundles2
```
Comment (vitalii427) [inline apiary.apib:2434]: Bundle2 more appropriate

File: apiary.apib [hunk 2]
```diff
@@ -1471,15 +1477,47 @@ __Warning__ Message sender needs to have "create_event" permission
 
     The createdBy user is not registered or has no "create_event" permission
 
-    + Body    
+    + Body
 
 ## Group Bundles
```
Comment (vitalii427) [inline apiary.apib:2238]: Bundle

File: apiary.apib [hunk 3]
```diff
@@ -1206,7 +1212,8 @@ Fetches an event by identifier
     Event not found
 
     + Body
-    
+
+
 ## Group Events2
```
Comment (vitalii427) [inline apiary.apib:1975]: Event2

File: apiary.apib [hunk 4]
```diff
@@ -614,15 +1801,208 @@ __Warning__ Request sender needs to have "create_asset" permission
 
     + Body
 
-### /assets/{assetId}/events?{fromTimestamp,toTimestamp,perPage,page,createdBy,data}
+## Group Events
```
Comment (vitalii427) [inline apiary.apib:1804]: Group Event & /event

**Comment by vitalii427** [comment]
Missing endpoints:

**Comment by vitalii427** [comment]
Use grep to find all endpoints:
`grep -r -A1 '@controller('|grep "'"
grep -r -A1 '@http'|grep "'"`

**Comment by SigismundSchlomo** [comment]
There is a complementary documentation on confluence
https://inc4net.atlassian.net/wiki/spaces/AMB/pages/175866027/Ambrosus+node+api
Also, there is a completed documentation (old link works too)
https://amrosusnode.docs.apiary.io/#introduction/serialization-[serialization]

File: apiary.apib [hunk 5]
```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
...
+                    },
+                },
+
+                "meta": {
+                    "code": 200
+                }
+            }
+
+
+## /restoreconfig
```
Comment (vitalii427) [inline apiary.apib:3925]: /admin/restoreconfig ?

File: apiary.apib [hunk 6]
```diff
@@ -1036,15 +1796,144 @@ __Warning__ Request sender needs to have "create_asset" permission
 
     + Body
 
-## Group Events
+## /exists/{assetId}
```
Comment (vitalii427) [inline apiary.apib:1799]: /asset2/exists/{assetId} ?

File: apiary.apib [hunk 7]
```diff
@@ -457,7 +939,7 @@ Fetches asset with provided assetId in the system.
     + Body
 
 ### /assets{?perPage,page,createdBy,fromTimestamp,toTimestamp,identifier}
```
Comment (vitalii427) [inline apiary.apib:941]: Seems that there's no such endpoint

**Comment by vitalii427** [comment]
/bundle2/exists/{bundleId} ?

**Comment by vitalii427** [comment]
Seems that there's no /events/...

**Comment by vitalii427** [comment]
/metrics/forceError

File: apiary.apib [hunk 8]
```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
...
+
+## /organization/invite/resend
+__Warning__ super_account permission is required
+#### Resend all invitation [GET]
++ Response 200 (application/json)
+
+    + Body
+
+
+## /organization/invite/
```
Comment (vitalii427) [inline apiary.apib:4638]: Same as /organization/invite{?perPage,page,createdBy,data}  ?

File: apiary.apib [hunk 9]
```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
...
+## Group Organization Request
+Request is the request to join organization. Consists of:
+* address - string, required - the account address that requests to join.
+* title - string, required - the title of organization that is requested
+* email - string, required - the email of requet sender
+* message - string, required - the message to organization
+* createdOn - number, required - the unix timestamp of time when request was created
+* refused - boolean, required - the status of request
+
+## /organization/request
```
Comment (vitalii427) [inline apiary.apib:4668]: Same as /organization/request/ ?

File: apiary.apib [hunk 10]
```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
...
+                },
+                "pagination": {
+                    "hasNext": false,
+                    "next": "eyIkb2lkIjoiNjE1ZWFjN2UzNjZhNTkwMDFiY2I1Y2VjIn0",
+                    "hasPrevious": false,
+                    "previous": "eyIkb2lkIjoiNjE1ZWFjN2UzNjZhNTkwMDFiY2I1Y2VjIn0"
+                }
+            }
+
+## /organization/update/{organizationId}
```
Comment (vitalii427) [inline apiary.apib:4314]: Seems no /organization/update/{organizationId}

**Comment by vitalii427** [comment]
I haven't checked endpoint parameters and description. Check 'em carefully pls

File: apiary.apib (hunk 7 above)
Comment (SigismundSchlomo) [reply apiary.apib:941]: There is an endpoint to use with Atlas. This doc is for both Hermes and Atlas

File: apiary.apib (hunk 9 above)
Comment (SigismundSchlomo) [reply apiary.apib:4668]: There is GET and POST calls description. I'll try to make it unambiguous

File: apiary.apib (hunk 8 above)
Comment (SigismundSchlomo) [reply apiary.apib:4638]: There is GET and POST calls description. I'll try to make it unambiguous

**Comment by SigismundSchlomo** [comment]
Parameters and description was proofread and some fixes were made. I hope, now this document is accurate. 
https://amrosusnode.docs.apiary.io/#reference/assets/assetsassetidevents

**Comment by vitalii427** [comment]
/assets - availiable only for Atlas node

File: apiary.apib [hunk 11]
```diff
@@ -368,49 +560,346 @@ It is not possible to modify protected accounts and accounts belonging to differ
...
+Assets are primary elements moving through a supply chain. The are the nouns of the system. They can represent an ingredient, product, package of products or any other type of container.
+
+An Asset on its own function only as a handle for a Event stream. Because of this, they only contain an `idData` structure holding minimal information about their creation.
+
+Assets are identified by an `assetId` field which is a sha3 hash calculated from the `content` field. \
+
+
+
+### /assets/{assetId}
+__ⓘ Note:__  Available for both Hermes and Atlas nodes.
```
Comment (vitalii427) [inline apiary.apib:877]: only Atlas

File: apiary.apib [hunk 12]
```diff
@@ -1377,220 +2147,2555 @@ Fetches an event by identifier
...
+                },
+                "pagination": {
+                    "hasNext": false,
+                    "next": "eyIkb2lkIjoiNjFhNGE4MWNmNjE2YmEwMDNjY2IyOGMwIn0",
+                    "hasPrevious": false,
+                    "previous": "eyIkb2lkIjoiNjFhNGE4MWNmNjE2YmEwMDNjY2IyOGMwIn0"
+                }
+            }
+
+## /{bundleId}/restore
```
Comment (vitalii427) [inline apiary.apib:3347]: /bundle/{bundleId}/restore

File: apiary.apib [hunk 13]
```diff
@@ -1206,17 +1988,15 @@ Fetches an event by identifier
     Event not found
 
     + Body
-    
-## Group Events2
 
+## Group Event
+__ⓘ Note:__ Available only for Hermes node. Lagacu endpoint event2 should be used.
```
Comment (vitalii427) [inline apiary.apib:1993]: One more misspell - 'Lagacu' )  Did you use a tool to spellcheck?

Comment (SigismundSchlomo) [reply apiary.apib]: File was checked with spellchecker now.

**Comment by vitalii427** [comment]
I'm done )

**Review by vitalii427 — APPROVED** [review]

**Comment by vitalii427** [comment]
https://amrosusnode.docs.apiary.io/

File: apiary.apib [hunk 14]
```diff
@@ -30,7 +30,7 @@ When serialising data for hashing and signing, special care needs to be taken so
 
 ## Hashing
 
-Hashes are a very important part of proving the persistence and the immutability of the data. All hashes in the AMB-NODE are computed using the ethereum standard `hashMessage` function as described in [web3 docs](https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#hashmessage). The underlying hash function is [Keccak256](https://keccak.team/index.html). 
+Hashes are a very important part of proving the persistence and the immutability of the data. All hashes in the AMB-NODE are computed using the ethereum standard `hashMessage` function as described in [web3 docs](https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#hashmessage). The underlying hash function is [Keccak256](https://keccak.team/index.html).
 
 ### Errors
```
Comment (vitalii427) [inline apiary.apib:35]: Decrease ident - ##

**Comment by SigismundSchlomo** [comment]
> https://amrosusnode.docs.apiary.io/ auto updates?

After merge old link will work 
https://ambrosus.docs.apiary.io/#introduction/

**Comment by vitalii427** [comment]
Merge it )

**Pull request merged**